### PR TITLE
ROX-13929: Add log to debug VM GraphQL test error

### PR DESCRIPTION
**Pull request by theencee**
## Description

Unable to repro this locally and it has only happened twice, so print out the actual error from the response for next time.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why
you did not do so. Valid reasons include, for example, "CI is sufficient",
"No testable changes". Feel free to attach JSON snippets, curl commands,
screenshots.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

**Review by keyallis — APPROVED** [review]

File: qa-tests-backend/src/test/groovy/VulnMgmtTest.groovy [hunk 1]
```diff
@@ -256,7 +256,14 @@ fragment cveFields on ImageVulnerability {
 
         def embeddedImageRes = gqlService.Call(getEmbeddedImageQuery(),
                 [id: imageDigest, query: "CVE:CVE-2017-10684"])
-        assert embeddedImageRes.hasNoErrors()
+
+        // Expanded instead of using hasErrors() for easier debugging if there are errors
+        // as the test framework will actually print out the errors now
+        assert embeddedImageRes.code == 200
```
Comment (md2119) [inline qa-tests-backend/src/test/groovy/VulnMgmtTest.groovy:262]: Is the concern only with the above graphQL call?

**Review by md2119 — APPROVED** [review]

File: qa-tests-backend/src/test/groovy/VulnMgmtTest.groovy (hunk 1 above)
Comment (theencee) [reply qa-tests-backend/src/test/groovy/VulnMgmtTest.groovy:262]: This is the one that has failed. Others might, but I'm hoping given this one has for sure we can get logs out of it.

**Comment by theencee** [comment]
The EKS failure seems to be unrelated

**Pull request merged**